node/full.go

```diff
@@ -1,6 +1,8 @@
 package node
 
 import (
+	coreclient "github.com/celestiaorg/celestia-node/core"
+	"github.com/celestiaorg/celestia-node/service/block"
 	"go.uber.org/fx"
```

Comment:
I would recommend to put our deps last:
```suggestion
	"go.uber.org/fx"
	
	coreclient "github.com/celestiaorg/celestia-node/core"
	"github.com/celestiaorg/celestia-node/service/block"

```

Reply:
I think this could even be caught by a linter. Maybe too many lints were turned off? (cc @Bidon15)